### ISSUE TITLE
Look into ways of improving logs for users running a node

### ISSUE THREAD

**Issue by hstove** [stacks-network/stacks-core#4729]
**This is a placeholder issue, more work needs to be done to figure out exactly what work should be done.**

For many people who run a node for the first time, there are a lot of logs, and often those logs make it seem like something is broken even when everything is behaving normally. In some cases this is due to a `warn` used when a `debug` might be better, or `info` for something that should maybe be a `debug`. There are also instances where a specific line is logged much too frequently.

Improving logs could make it easier to run a node and better debug when things go wrong.

**Comment by AshtonStephens** [comment]
Is this ticket consensus critical for Nakamoto?

**Comment by hstove** [comment]
@AshtonStephens nope, it's just to improve signer onboarding

**Comment by hstove** [comment]
One small change we could make it to support different log levels. Right now, you can only use either `trace`, `debug`, or `fatal` log levels specified via ENV vars. We could also support `warn` and `error` by changing this code:

https://github.com/stacks-network/stacks-core/blob/ae5b71673492426561c9212af5d997ca93966256/stacks-common/src/util/log.rs#L247-L259

**Comment by AshtonStephens** [comment]
Yeah actually that would be a great fast improvement, I see we have `warn!` a lot in the code.

**Comment by saralab** [comment]
To Do: Convert this to a discussion
